pkg/injector/patch_test.go

```diff
@@ -106,16 +129,23 @@ func TestCreatePatch(t *testing.T) {
 				nonInjectNamespaces: mapset.NewSet(),
 			}
 
+			if tc.os == "windows" {
+				mockConfigurator.EXPECT().GetEnvoyImage().Return("").Times(1)
+				mockConfigurator.EXPECT().GetEnvoyWindowsImage().Return("").Times(1)
+			} else {
+				mockConfigurator.EXPECT().GetEnvoyImage().Return("").Times(1)
+				mockConfigurator.EXPECT().GetEnvoyWindowsImage().Return("").Times(0)
+			}
```

Comment:
To avoid such conditionals with mocks, initialize the mock object within the loop for each test case in the table of tests and set both to `AnyTimes()`

Reply:
fixed in a8f1df1ea78e0932a6f67f3c51cdb6c6bbf8eb8f